substra/cli/interface.py

```diff
@@ -491,6 +531,47 @@ def add_traintuple(ctx, objective_key, algo_key, dataset_key, data_samples, in_m
     printer.print(res, is_list=False)
 
 
+@add.command('aggregate_traintuple')
+@click.option('--objective-key', required=True)
+@click.option('--algo-key', required=True)
+@click.option('--in-model-key', 'in_models_keys', type=click.STRING, multiple=True,
```

Comment:
For an aggregate traintuple, these in model keys are mandatory I think.

Reply:
I'm not sure we should force input models. We could create an aggregate without input models (kind of initialization step).